Applications/Spreadsheet/SpreadsheetWidget.cpp

```diff
@@ -175,6 +175,11 @@ void SpreadsheetWidget::load(const StringView& filename)
         GUI::MessageBox::show_error(window(), result.error());
         return;
     }
+
+    m_tab_widget->on_change = nullptr;
+    m_cell_value_editor->on_change = nullptr;
+    m_current_cell_label->set_text("");
+    m_should_change_selected_cells = false;
```

Comment:
This sort of thing is always gnarly and it's nice to solve with e.g `WeakPtr` when possible.
I don't mind doing it this way, just lamenting that it's a bit of a bitch :)

Reply:
Hmmm...weak pointer to a function?